microsoftgraph/microsoft-graph-explorer-v4
# fix/Enable screen reader confirmation feedback

## Overview

Screen reader conveys confirmation feedback after successfully :
- adding 'Key' and 'Value' to the table present in the Request headers tab.
- deleting 'Key' and 'Value' on the table present in the Request headers tab.
- activating and deactivating 'Minimize sidebar button'
- activating the 'Expand' button in the Response area tab items.

fixes #796 

## Testing Instructions

1. Launch Narrator.
2. Open the URL in Anaheim edge dev browser.
3. Navigate to the 'Request headers' tab present in the right side of the pane and activate it. 'Request headers' section will open.
4. Navigate and fill the required details in the 'Key' and 'Value' edit box.
5. Navigate to the 'Add' button and activate it using enter key. Values of the 'Key' and 'Value' edit boxes gets successfully added in the table.
6. Observe the screen reader message.
7. Observe similar response upon deletion of the 'Key' and 'Value' Item.
8. Observe feedback on activating the 'Minimize sidebar button' and 'Expand' button
